### ISSUE TITLE
Next & Previous Issues while in Modal

### ISSUE THREAD

**Issue by jefforshalick** [ZenHubIO/support#300]
When I'm in the modal view, it would be nice to be able to flip the next or previous issue in the pipeline without having to close the modal. It's hard to keep track of where you are all the time. #287 might help with this but not having to close the modal would help a lot.

**Comment by aupright** [comment]
Thanks for the suggestions @jefforshalick! I can see where this would be a great way to familiarize yourself with new (or recently added) issues without having to navigate away from the boards page. Let's leave this one open in the discussion column to see if we can get a few more votes from the community.

**Comment by talsafran** [comment]
Totes +1.

Maybe include a key at the bottom with the titles of the previous and next stories?

---

cc @zeke @joyjding

**Comment by joyjding** [comment]
Yes, this would definitely be useful, especially if we could use keyboard shortcuts (left and right, more likely, up and down) to move through issues.

**Comment by wires** [comment]
:+1: for this, esp. including labels and keyboard shortcuts; also it should be *fast*. :car::dash: We did a sprint poker session on the beamer today, this feature would be very pleasant. Eg. when trying to find a particular issue, most issues contained screenshots/`formatted code`, making them quick to spot visually. (we essentially mimicked this feature by opening a gazillion browser tabs)
